kubetest2-gce/ci-tests/buildupdown-legacy.sh

```diff
@@ -27,8 +27,6 @@ cd "${GOPATH}/src/k8s.io/kubernetes"
 # kubetest2 against k/k
 kubetest2 gce \
     -v=2 \
-    --repo-root=. \
```

Comment:
I'm going to revert the change to this file and create a prowjob for the new script in this folder that specifically tests prebuilt k/k scenarios.